pkg/operator/ceph/object/admin.go

```diff
@@ -67,7 +67,7 @@ func RunAdminCommandNoMultisite(c *Context, args ...string) (string, error) {
 	}
 
 	// start the rgw admin command
-	output, err := c.Context.Executor.ExecuteCommandWithTimeout(timeout, command, args...)
+	output, err := c.Context.Executor.ExecuteCommandWithTimeout(timeout, true, command, args...)
```

Comment:
I think you can instead call `ExecuteCommandWithOutputFile()` to have the output written to a file so the stdout and stderr won't conflict. Does that work for you? Then we wouldn't need the new flag.

Reply:
Oh, good idea, I'll fix!

Reply:
Unfortunately this function can't be used because `radosgw-admin` doesn't have an option to specify the output file. I introduced a new function that returns only stdout and can specify timeout.

Reply:
I found that extracting json from combined output seems to be better. Some error handling  routine s use whether the output contains some strings, to judge the type of errors. I'll fix my PR later.

Reply:
@satoru-takeuchi How did you repro this issue? I heard another instance of it for an external cluster scenario, but I can't get a repro yet. Anyway, looking forward to the fix, thanks.

Reply:
This issue happens in the CI of my k8s cluster. Then OB is created under heavy load consists of RBD creation and many applications' initialization. However, I don't know the reliable simple reproducer yet.